src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RisksAndNeedsService.kt

```diff
@@ -67,6 +67,7 @@ class RisksAndNeedsService(
   fun createSupplementaryRisk(referralId: UUID, crn: String, user: AuthUser, riskCreatedAt: OffsetDateTime, riskInformation: String, redactedRisk: RedactedRisk? = null): UUID {
 
     val request = if (canPostFullRiskInformation && redactedRisk != null) {
+      logger.debug("Sending full supplementary risk information to ARN")
```

Comment:
Would this be possible to monitor via AppInsights and queries that specifically look for dependency calls?

Context: I wanted to initially ask if this would be useful in other environments -- then wanted to ask if it's worth having a custom metric for this -- then realised dependency calls are tracked in AppInsights